components/cdc/src/delegate.rs

```diff
@@ -993,113 +1000,4 @@ mod tests {
         assert!(err.has_epoch_not_match());
         assert!(err.take_epoch_not_match().current_regions.is_empty());
     }
-
-    #[test]
-    fn test_scan() {
```

Comment:
Should we add this test back later?

Reply:
It tests `on_scan` but it is removed in this PR, so I think we can just remove the test too.